imcsdk/apis/admin/communication_services.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This module implements all the communication services
+"""
+from imcsdk.mometa.comm.CommIpmiLan import CommIpmiLan, CommIpmiLanConsts
+
+
+def enable_ipmi(handle, priv=CommIpmiLanConsts.PRIV_ADMIN, key='0'*40):
+    """ Enable IPMI over LAN
+
+    :param priv: Optional privilege level: 'admin', 'user', 'read-only'
```

Comment:
the docstring format we follow is slightly different.. can you look it up in the other files and try to use the same.